WordPress/Classes/ViewRelated/NUX/WordPressAuthenticationManager.swift

```diff
@@ -446,6 +446,7 @@ extension WordPressAuthenticationManager: WordPressAuthenticatorDelegate {
             return true
         }
 
+        return false
```

Comment:
[Q] I'm wondering if you added this while testing? I think we still want to return false only when a wp.com account has no sites
```suggestion
```